test/README.md

```diff
@@ -215,13 +215,6 @@ Change directory to the test directory:
 cd test
 ```
 
-To setup up minikube for drenv run once before starting any
-environment:
```

Comment:
Here we should remove "minikube", since we support multiple providers (default to lima on macOS and minikube on Linux).